timemachine/exchange/exchange_mover.py

```diff
@@ -61,12 +63,15 @@ def build_system(host_pdbfile: str, water_ff: str, padding: float):
     return solvated_host_system, solvated_host_coords, box, solvated_topology, nwa
 
 
-# currently un-used.
 def randomly_rotate_and_translate(coords, offset):
+    # coords: shape 3(OHH) x 3(xyz), water coordinates
     centroid = np.mean(coords, axis=0, keepdims=True)
     centered_coords = coords - centroid
```

Comment:
point of uncertainty: sensitivity of MC acceptance rate to choice of pivot

(rotate around oxygen, vs. rotate around centroid, vs. rotate around center of mass)

Reply:
not sure how much this matters in practice

Reply:
Would guess not very sensitive, but I don't know -- I don't remember if there was a big difference between "translate only" vs. "translate + randomly rotate"? If so, might also be sensitive to little details like this of how exactly the proposal is prepared.

I guess, if uncertain, we could also randomize the choice of pivot (weighted centroid, random weights).

Reply:
For biased deletions you need to randomly rotate, this is because you're only picking 1 or 2 unique waters with fixed orientations. In order to successfully insert into the buckyball, it's unlikely the original orientation is "aligned" with whatever environment is in the buckyball.